style.css

```diff
@@ -308,3 +314,102 @@ button:disabled {
   font-size: 12px;
   color: #b3bac5;
 }
+
+@media screen and (min-width: 992px) {
+  p {
+    font-size: 20px;
+  }
+
+  .header {
+    background-image: url("./assests/desktop-header-bg.png");
+    background-position-y: 0;
+    padding: 140px 160px 0 160px;
+  }
```

Comment:
Nice work implementing most of the project requirements. You just have one more thing to do to get your project approved.

- [ ] The `background-image` used in the header section should be the same as that used in Figma Template. Currently, this is what you have:

![image](https://user-images.githubusercontent.com/77106962/144279915-228f2f06-3d49-47d0-a4c4-93cfc8ab097d.png)
Whereas, Figma's template's is:

## Figma

![image](https://user-images.githubusercontent.com/77106962/144280372-6a777106-1d64-4399-94c6-205e89bd7158.png)

#### I think it would be nice to make it look exactly like that of Figma's 👍